src/Layer/ColorLayer.js

```diff
@@ -73,6 +73,15 @@ class ColorLayer extends Layer {
     convert(data, extentDestination) {
         return textureConverter.convert(data, extentDestination, this);
     }
+
+    /**
+    * All layer's textures are removed from scene and disposed from video device.
+    */
+    removeDisposeNodes() {
```

Comment:
Maybe add a "fake" method in `Layer` as well: either doing nothing, or printing a warning that this method needs to be implemented (like in `Source`).

Reply:
Also this seems like a weird name to understand

Reply:
done

Reply:
I rename to `Layer.delete`